_assets/patches/geth/README.md

```diff
@@ -16,13 +16,23 @@ Instructions for creating a patch from the command line:
     1. If you already have a commit that represents the change, find its SHA1 (e.g. `$COMMIT_SHA1`) and do `git diff $COMMIT_SHA1 > file.patch`
     1. If the files are staged, do `git diff --cached > file.patch`
 
+# Updating patches
+
+1. Remove tweak the patch file.
+2. Run `make dep-ensure` to re-apply patches.
```

Comment:
Nitpicking: Like above let markdown do the numbering. 😬

Reply:
ah, I tend to look at md files as sources, not after rendering...

Reply:
fixed for consistency reasons